config/prometheus/monitor.yaml

```diff
@@ -1,16 +1,20 @@
-
 # Prometheus Monitor Service (Metrics)
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-  name: controller-manager-metrics-monitor
-  namespace: system
+  name: capsule-monitor
+  namespace: capsule-system
 spec:
   endpoints:
-    - path: /metrics
+    - interval: 15s
+      path: /metrics
       port: metrics
+  jobLabel: app.kubernetes.io/name
+  namespaceSelector:
+    matchNames:
+    - capsule-system
   selector:
     matchLabels:
-      control-plane: controller-manager
+      app.kubernetes.io/name: capsule
```

Comment:
We have to use the labels provided by kustomize, not the Helm ones, so `control-plane=controller-manager` should be good.

Reply:
checked "capsule/config/manager/metrics_service.yaml" and found :

`  labels:
    control-plane: controller-manager`

so, as you said, it should be good to revert this line